novm/manager.py

```diff
@@ -227,7 +227,7 @@ def state(output):
                 index=1+len(nics)+len(disks),
                 pci=not(nopci),
                 tag="root",
-                tempdir=self._instances.file(str(os.getpid())),
+                tempdir=self._myInstance.file("writable"),
```

Comment:
Welcome back.

Yes. I will push a new commit to change log file from novmm.log to $PID.log.

And After you review, I will collapse all commits into one and resend a new PR.